### PR TITLE
@damassi => Add OtherAuctions section for auction version of artwork page

### PR DESCRIPTION
**Pull request by mzikherman**
Looks like:

screen shot 2018-12-13 at 2 43 19 pm

Definitely needs a bit of design sweep (making sure the links work), but prob mergeable. Maybe we can pair on a design sweep?

### CODE REVIEW

File: src/Apps/Artwork/Components/OtherWorks/ArtworkContexts/ArtworkContextAuction.tsx [hunk 1]
```diff
@@ -38,8 +40,8 @@ export const ArtworkContextAuctionQueryRenderer: React.SFC<
                 $excludeArtworkIDs: [String!]
                 $isClosed: Boolean!
               ) {
-                artwork(id: $artworkSlug) {
-                  ...ArtworkContextAuction_artwork
+                viewer {
```
Comment (mzikherman) [inline src/Apps/Artwork/Components/OtherWorks/ArtworkContexts/ArtworkContextAuction.tsx]: We needed to make this query spread on `Viewer`, and not `Artwork`, so we could also include the 'current sales' fragment.

File: src/Styleguide/Components/AuctionCard.tsx [hunk 2]
```diff
@@ -9,6 +10,36 @@ import {
   Sans,
   Serif,
 } from "@artsy/palette"
+import { createFragmentContainer, graphql } from "react-relay"
+
+const zone = time => {
+  return moment(time).tz("America/New_York")
+}
+
+const upcomingLabel = (
```
Comment (mzikherman) [inline src/Styleguide/Components/AuctionCard.tsx]: This was taken right from https://github.com/artsy/force/blob/d2d9e3a47f12b4c0ecda98f75e48b86cd9f13665/src/desktop/components/react/auction_block/utils/upcoming_label.js

**Comment by artsyit** [comment]
Deploy preview for *artsy-reaction* ready!

Built with commit b2b1b14257b059403d877faa527d2ac278e65a3b

https://deploy-preview-1695--artsy-reaction.netlify.com

File: src/Apps/Artwork/Components/OtherWorks/ArtworkContexts/__tests__/ArtworkContextAuction.test.tsx [hunk 3]
```diff
@@ -9,9 +9,7 @@ describe("ArtworkContextAuction", () => {
   const getWrapper = (props = ArtworkContextAuctionFixture) => {
     return mount(
       <RelayStubProvider>
-        <ArtworkContextAuctionFragmentContainer
-          artwork={props.artwork as any}
-        />
+        <ArtworkContextAuctionFragmentContainer viewer={props.viewer as any} />
```
Comment (mzikherman) [inline src/Apps/Artwork/Components/OtherWorks/ArtworkContexts/__tests__/ArtworkContextAuction.test.tsx]: I have no clue why this spec is failing (the error output is hard to grok). I updated the fixture and signature here to just reflect the nesting change to `viewer`.

File: src/Styleguide/Components/AuctionCard.tsx (hunk 2 above)
Comment (damassi) [reply src/Styleguide/Components/AuctionCard.tsx]: Is this data available on MP?

Comment (damassi) [reply src/Styleguide/Components/AuctionCard.tsx]: I'm not sure how worth it it is to have every message variation in MP (makes sense if these messages are universal, though, or used elsewhere. That said if the logic is roughly identical to what is currently in MP maybe there's a way to share?

**Pull request merged**